pkg/server/container_start_test.go

```diff
@@ -309,23 +309,37 @@ func TestGenerateContainerMounts(t *testing.T) {
 		securityContext *runtime.LinuxContainerSecurityContext
 		expectedMounts  []*runtime.Mount
 	}{
-		"should setup ro /etc/hosts mount when rootfs is read-only": {
+		"should setup ro mount when rootfs is read-only": {
 			securityContext: &runtime.LinuxContainerSecurityContext{
 				ReadonlyRootfs: true,
 			},
-			expectedMounts: []*runtime.Mount{{
-				ContainerPath: "/etc/hosts",
-				HostPath:      testSandboxRootDir + "/hosts",
-				Readonly:      true,
-			}},
+			expectedMounts: []*runtime.Mount{
+				{
+					ContainerPath: "/etc/hosts",
+					HostPath:      testSandboxRootDir + "/hosts",
+					Readonly:      true,
+				},
+				{
+					ContainerPath: resolvConfPath,
+					HostPath:      getResolvPath(testSandboxRootDir),
```

Comment:
nit: Use `testSandboxRootDir + "/resolv.conf"`, because we don't know whether `getResolvPath` works or not, which is also what we want to test.